src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+function cloneColumn(columnIndex) {
```

Comment:
The function `cloneColumn` should accept only one parameter `columnIndex`, but in line 24, it is called with two arguments. This is an error related to the task requirements.